terraform/hcl/applicable/req_sw_terraform_aws_athena_encrypt_workgroup.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_athena_workgroup" "vulnerable_example" {
+    name = "dummy"
+    # encryption_configuration is not set
+}
```

Comment:
<div align="center"> 

### 🛠️ Infrastructure as Code Vulnerability
	
| Severity | Finding |
| :---: | :---: |
| ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableMediumSeverity.png)<br> | AWS Athena workgroup does not usedata-at-rest encryption |

</div>

<details>
<summary> <b>Full description</b> </summary>
<br>

Resource `aws_athena_workgroup` should have key `encryption_configuration`

Vulnerable example - 
```
resource "aws_athena_workgroup" "vulnerable_example" {
      # encryption_configuration is not set
}
```

Secure example -
```
resource "aws_athena_workgroup" "secure_example" {
  configuration {
    result_configuration {
      encryption_configuration {
        encryption_option = "SSE_KMS"
        kms_key_arn       = aws_kms_key.example.arn
      }
    }
  }
}
```

</details>


---
<div align="center">

[🐸 JFrog Frogbot](https://github.com/jfrog/frogbot#readme)

</div>